packages/terra-alert/tests/wdio/alert-spec.js

```diff
@@ -72,6 +72,22 @@ Terra.describeViewports('Alert', ['tiny', 'large'], () => {
   });
 
   describe('Alert Long Text', () => {
+    it('focuses on show more button', () => {
+      browser.url('/raw/tests/cerner-terra-core-docs/alert/long-text-alert');
+      browser.keys(['Tab']);
+      expect($('button.button').isFocused).toBeTruthy();
+      Terra.validates.element('show more button focused');
+    });
+
+    it('open show hide upon button click', () => {
+      $('button.button').click();
+      Terra.validates.element('disclosed show hide');
+    });
+
+    it('close show hide', () => {
+      $('button.button').click();
```

Comment:
I think validation that the show hide was closed could be added here as well

Reply:
Added here: https://github.com/cerner/terra-core/pull/3889/commits/20269f6fbdba9a65a3f4a2ad03501a2a069a200b